app/config/featureswitch/FeatureSwitch.scala

```diff
@@ -80,3 +80,8 @@ case object DeductionBreakdown extends FeatureSwitch {
   override val name: String = s"$prefix.enable-calculation-deduction-breakdown"
   override val displayText: String = "Enable Deduction Calc Breakdown"
 }
+
+case object TaxDue extends FeatureSwitch {
```

Comment:
Please also add this FS into line 37 after the DeductionBreakdown, otherwise, the new FS will not show in the test-only feature-switches list

val switches: Set[FeatureSwitch] = Set(
    Payment,
    Estimates,
    Bills,
    ReportDeadlines,
    ObligationsPage,
    IncomeBreakdown,
    DeductionBreakdown
  )

Reply:
Done